### PR TITLE
feat: improved metrics

### PR DESCRIPTION
**Pull request by hassannaveed1997**
## Description
When we get metrics for a model, they are hard to put into context. There were two problems:
- MAE: the numbers aren't intuitive by themselves
- MAPE: since the targets were scaled, we had values very close to zero so this wasn't numerically stable. Our error percentages could be 10**8 or something along these lines

## Solution
I got rid of MAPE entirely, since this was no longer applicable.
To put MAE into context, it is now compared against a benchmark. Here the benchmark model was just a mean, but can later be improved to be another model that inherits properties from the base model. The new section is shown below:
image

### CODE REVIEW

File: wod_predictor/feature_engineering_parts/helpers.py
Comment (qzheng75) [inline wod_predictor/feature_engineering_parts/helpers.py:1]: No comments with this file. LGTM!

File: wod_predictor/modeling.py [hunk 1]
```diff
@@ -54,7 +46,29 @@ def show_results(self, y_test, meta_data={}):
 
             show_breakdown_by_workout(y_pred_unstacked, y_test_unstacked)
 
+        self.get_comparative_stats()
+
+    # get comparative stats
+    def get_comparative_stats(self):
+        baseline_model = BaselineModel()
```
Comment (qzheng75) [inline wod_predictor/modeling.py:53]: Just suggesting an additional feature: can we allow users to use an arbitrary model (as long as it inherits from BaseModeler) as the baseline model?

File: wod_predictor/models/helpers.py
Comment (qzheng75) [inline wod_predictor/models/helpers.py:1]: It works after testing with the pipeline. Great!

**Review by qzheng75 — APPROVED** [review]
Everything works. The only comment is in modeling.py on a potential additional feature.

File: wod_predictor/modeling.py (hunk 1 above)
Comment (hassannaveed1997) [reply wod_predictor/modeling.py:53]: Thats actually a good idea, will skip implementing it for now but adding to the backlog.

File: wod_predictor/models/helpers.py
Comment (hassannaveed1997) [reply wod_predictor/models/helpers.py:1]: Thanks for testing!

**Pull request merged**